utils/create_fixture.py

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#   Quan Zhou <quan@bitergia.com>
+#
+
+
+import argparse
+from datetime import datetime, timezone
+import json
+import logging
+
+import pymysql
```

Comment:
Instead of `pymysql` we should use `mysqlclient`, which is already a sortinghat dependency.